boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -60,34 +59,171 @@ currently supports the following hardware features:
    * - GPIO
      - on-chip
      - gpio
+   * - ADC
+     - on-chip
+     - adc
+   * - I2C
+     - on-chip
+     - i2c
+   * - KSCAN
+     - on-chip
+     - kscan
+
+
+Other hardware features are not currently supported by Zephyr.
+
+The default configuration can be found in the
+:zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
+
+Programming and debugging on it83202
+************************************
 
-Programming and debugging
-*************************
+In order to upload the application to the device,
+you'll need our flash tool and Download board.
+You can get them at:`ITE's website`_.
```

Comment:
This is only required for Windows users, right?

If so, add that...

> This tool is only required for Windows users. For Linux users, `west flash` can flash the board without additional tooling.

Reply:
Yes. And maybe I need to find a place to store the linux version flash tool.